pkg/logql/marshal/legacy/marshal_test.go

```diff
@@ -145,7 +177,7 @@ func Test_MarshalTailResponse(t *testing.T) {
 
 func Test_QueryResponseMarshalLoop(t *testing.T) {
 	for i, queryTest := range queryTests {
-		var r loghttp.QueryResponse
+		var r map[string]interface{}
```

Comment:
I'm a bit confused why this needs to be `map[string]interface{}`. Don't we need to add the `stats` struct to the `loghttp.QueryResponse`?

Reply:
This object is used only in the test.  Kinda confusing yes, I think there some clean up todo. There were no point using it here cause that's not what is being used.

Reply:
We should remove that struct. But anyway this is legacy.